docs/docs/end-users/Rider.md

```diff
@@ -6,56 +6,16 @@ index: 7
   
-From Rider 2022.2 onwards, Rider can detect your `dotnet` Fantomas installation, either globally or locally.  
-Install Fantomas locally with `dotnet tool install fantomas`.
-
-<img class="mt-2" src="{{root}}/images/rider-fantomas.png" alt="drawing" width="70%"/>
-
-At the time of writing, Rider versions prior to the not yet released 2022.3 [overwrite](https://youtrack.jetbrains.com/issue/RIDER-83997/Rider-doesnt-respect-Fantomas-default-settings-not-explicitly-set-in-editorconfig)
-the settings of Fantomas which aren't explicitly set in your editorconfig file.  
-As the settings chosen by Rider can lead to unwanted results, Rider users might want to also include the default settings in their editorconfig file.
-
-Fortunately, with 2022.3 the Rider developers changed the settings chosen by Rider to be the same as the default Fantomas settings.  
-This means, as long as the Fantomas default settings don't change, the described issue should be solved with Rider 2022.3.  
-The current default settings are the following:
+From Rider 2022.2 onwards, Rider can detect your `dotnet` Fantomas installation, either globally or locally.
 
+Install Fantomas locally with:
 ```
-indent_size=4
-max_line_length=120
-end_of_line=crlf
-insert_final_newline=true 
-fsharp_space_before_parameter=true 
-fsharp_space_before_lowercase_invocation=true 
-fsharp_space_before_uppercase_invocation=false
-fsharp_space_before_class_constructor=false
-fsharp_space_before_member=false
-fsharp_space_before_colon=false
-fsharp_space_after_comma=true 
-fsharp_space_before_semicolon=false
-fsharp_space_after_semicolon=true 
-fsharp_space_around_delimiter=true 
-fsharp_max_if_then_short_width=0
-fsharp_max_if_then_else_short_width=60
-fsharp_max_infix_operator_expression=80
-fsharp_max_record_width=40
-fsharp_max_record_number_of_items=1
-fsharp_record_multiline_formatter=character_width
-fsharp_max_array_or_list_width=80
-fsharp_max_array_or_list_number_of_items=1
-fsharp_array_or_list_multiline_formatter=character_width
-fsharp_max_value_binding_width=80
-fsharp_max_function_binding_width=40
-fsharp_max_dot_get_expression_width=80
-fsharp_multiline_bracket_style = cramped
-fsharp_newline_between_type_definition_and_members=true 
-fsharp_align_function_signature_to_indentation=false
-fsharp_alternative_long_member_definitions=false
-fsharp_multi_line_lambda_closing_newline=false
-fsharp_experimental_keep_indent_in_branch=false
-fsharp_blank_lines_around_nested_multiline_expressions=true 
-fsharp_bar_before_discriminated_union_declaration=false
-fsharp_keep_max_number_of_blank_lines=100
-fsharp_strict_mode=false
+dotnet tool install fantomas
 ```
 
+<img class="mt-2" src="{{root}}/images/rider-fantomas.png" alt="drawing" width="70%"/>
+
+Prior to Rider 2022.3 it did not [respect](https://youtrack.jetbrains.com/issue/RIDER-83997/Rider-doesnt-respect-Fantomas-default-settings-not-explicitly-set-in-editorconfig) the default settings of Fantomas.
```

Comment:
Just a fun story to share: at some point Rider took the actual defaults from Fantomas and hardcoded them to its settings. Later Fantomas has changed the defaults, so the ones in Rider have become outdated, and for some time we were unfortunately stuck with them. It was never an intention to not respect the settings, just a coincidence of how the things have evolved here 🙂

Reply:
I think that a lot of software evolve like that :) A "good enough" solution is often the best starting point, and then when/if it breaks it is easier to design the "good" solution (more knowledge available).

But Rider sure does have some fun formatting quirks :) It is pretty confusing to figure out what settings it applies and when. E.g. when I edit a `.fsproj` file it sort of uses a mix of the "code style" settings for XML-files (specified in the Settings UI), and the settings I have specified in `.editorconfig` – but this is not shown anywhere in settings UI (i.e. which settings are overriden by `.editorconfig` – or the way around). And then when Rider modifies `.fsproj` it uses a whole different set of formatting rules :)

Reply:
Oh, the `.fsproj` story is also interesting. When editing the file manually then IntelliJ rules are used, but when Rider modifies it from code it actually delegates a lot of the work to MSBuild and it doesn't seem to follow what the rest of the IDE does. I'm not sure how easy/feasible it's to intercept the MSBuild modifications, but it seems it was difficult enough to become less prioritized.